pkg/consoleplugin/consolepluginserver_test.go

```diff
@@ -51,7 +52,7 @@ func TestConsolePluginServer(t *testing.T) {
 func waitForReady(t *testing.T) {
 	cl := http.Client{}
 
-	err := wait.Poll(DefaultRetryInterval, DefaultTimeout, func() (done bool, err error) {
+	err := wait.PollUntilContextTimeout(context.TODO(), DefaultRetryInterval, DefaultTimeout, false, func(ctx context.Context) (done bool, err error) {
```

Comment:
Poll is being deprecated and the linter was complaining. However to fix this there was a typo in the function name that was suggested as replacement, which was later fixed. 
PTAL here if the context is used correctly - I'm not a 100% sure.
For reference please check - https://github.com/kubernetes/apimachinery/issues/153 and the fix here - https://github.com/kubernetes/kubernetes/pull/117143/files